test/decorators.py

```diff
@@ -404,6 +404,20 @@ def metafunc(self, with_wasm64, *args, **kwargs):
   return metafunc
 
 
+def also_with_fetch_backend(f):
+  assert callable(f)
+
+  @wraps(f)
+  def metafunc(self, with_fetch, *args, **kwargs):
+    if with_fetch:
+      self.set_setting('FETCH_STREAMING', '2')
```

Comment:
Could you do `self.cflags.append('SKIP_SYNC_TESTS')` here maybe and save some complexity elsewhere?   I'm not sure.. maybe it not a shared pattern?